### PR TITLE
Internal: Editor V2 - History pacakge [ED-9735]

### PR DESCRIPTION
**Pull request by rotemee**

### CODE REVIEW

File: packages/packages/history/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@elementor/structure",
```
Comment (StyleShit) [inline packages/packages/history/package.json:2]:
```suggestion
  "name": "@elementor/history",
```

File: packages/packages/history/src/init.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import useActionProps from './hooks/use-action-props';
+
+export default function init() {
+	registerTopBarMenuItems();
+}
+
+function registerTopBarMenuItems() {
+	registerToggleAction( 'main', {
+		name: 'open-history',
+		priority: 20,
```
Comment (StyleShit) [inline packages/packages/history/src/init.ts:11]: check out the priorities I used in other PRs to make sure it works properly

File: packages/packages/history/src/init.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+		useProps: () => useActionProps(),
```
Comment (StyleShit) [inline packages/packages/history/src/init.ts:12]:
```suggestion
		useProps: useActionProps,
```

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: packages/packages/history/src/init.ts (hunk 2 above)
Comment (rotemee) [reply packages/packages/history/src/init.ts:11]: As agreed, will keep it as it is.

**Review by StyleShit — DISMISSED** [review]

**Review by Nevoss — DISMISSED** [review]

**Review by StyleShit — APPROVED** [review]

**Pull request merged**